test/src/csg/TestCSGCylindersMeshGenerator.C

```diff
@@ -0,0 +1,87 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "TestCSGCylindersMeshGenerator.h"
+#include "CSGBase.h"
+
+registerMooseObject("MooseTestApp", TestCSGCylindersMeshGenerator);
+
+InputParameters
+TestCSGCylindersMeshGenerator::validParams()
+{
+  InputParameters params = MeshGenerator::validParams();
+
+  params.addRequiredParam<std::vector<Real>>("radii", "list of radii for concentric cylinders");
+  params.addRequiredParam<Real>("height", "cylinder height");
+  params.addRequiredParam<std::vector<Real>>("center", "center point of cylinder");
+  params.addRequiredParam<std::string>("axis", "axis alignment");
```

Comment:
```suggestion
  MooseEnum axis("x y z");
  params.addRequiredParam<MooseEnum>("axis", axis, "axis alignment");
```

Reply:
FYI- I cannot get this to work. What does the input look like if you use MooseEnum like this? It compiles, but the tests fail to run because the input isn't right. I have tried: `axis = 'x'`, `axis = "x"`, and `axis = x`